tests/containers/podman_ipv6.pm

```diff
@@ -64,9 +67,9 @@ sub run {
     # Pull image from openSUSE registry (over IPv6 now)
     assert_script_run("podman pull $image", timeout => 300);
 
-    # Create the IPv6 network
+    # Create the IPv6 network and test container
     assert_script_run(sprintf('podman network create --ipv6 --subnet %s podman-ipv6', $subnet));
-    assert_script_run("podman run --name test-ipv6 --network podman-ipv6 --ip6 $cont_addr -d -p 80:80 -p '[::]:80:80' $image sleep 999", timeout => 180);
+    assert_script_run("podman run --name test-ipv6 --network podman-ipv6 --ip6 $cont_addr -d $image sleep 999", timeout => 180);
```

Comment:
Why don't we use a local network here? We can enable IPv6 for the default network or create a new one and then test the IPv6 connectivity from within the container without exposing a whole subnet.

Reply:
Done.